TODO.md

```diff
@@ -0,0 +1,7 @@
+TODO
+----
+
+- Implement a separate lambda function to monitor the events queue and perform validation
```

Comment:
I have added this, but feel free to remove it as in the readme we say "at some point we may do this".

Reply:
Maybe reword it to say " Implement a separate lambda function to perform schema validation"
We won't need one to monitor it, but we should perhaps design the xray monitoring around this with alerting in some way.

Reply:
Makes sense. I will change in a subsequent commit.